proto/redpanda/api/dataplane/v1alpha2/pipeline.proto

```diff
@@ -459,8 +459,14 @@ message GetPipelineServiceConfigSchemaRequest {
     // The service configuration is returned in JSON format.
     FORMAT_JSON = 2;
 
+    // The service configuration is returned in basic JSON format with metadata and examples.
+    FORMAT_JSON_FULL = 3;
```

Comment:
Do we need JSON and JSON FULL? what's the difference?

Reply:
This is described in the PR description.